comet_core/api_helper.py

```diff
@@ -69,3 +70,29 @@ def decorated(*args, **kwargs):
         return f(*args, **kwargs)
 
     return decorated
+
+
+def valid_token(fingerprint, token):
```

Comment:
also this name could be changed to validate_token or check_valid_token so it is clear that the token is being checked and not already checked.